lib/config.js

```diff
@@ -3,7 +3,7 @@
 export default {
   'shoebot-dir': {
       type:"string",
-      default:"~/.virtualenvs/shoebot/",
-      description: `Path where Shoebot was installed`,
+      default:"/usr/local/",
+      description: `Path where Shoebot was installed. If you installed Shoebot system-wide, this should be "/usr/local/". If you installed it with virtualenvwrapper, it's "~/virtualenvs/shoebot".`,
```

Comment:
`~/.local/` might be good - it's the default if you install with `pip --user` (and I think --user *is* the default).
trying to untangle dependencies installed via pip (like Pillow) from the users system installed ones isn't much fun.